### DISCUSSION TITLE
Now i can't create helpers

### DISCUSSION THREAD

**Discussion by aughtago** [PRProd/HA-Firemote#546] [Q&A] [answered]
I'm feeling really stupid tonight.  I searched all discussions for 'helper' with 0 results. Apparently, I am the only one stupid enough to have trouble with helpers.

I have created 9 helpers in the past, I now want to create a timer helper. I go to Helpers, 'add helper' and select the timer helper.  The dialog box opens, i fill in the fields, but when i check the "create" button, the dialog box goes away, and I still have only 9 helpers. it's not limited to timer, i tried a button helper too, same result. I verified that I have the Default_Config line in my configuration YAML.

**Comment by PRProd** [comment, marked as answer]
@aughtago ~

This is a discussion forum for Firemote, not a broader message board for general Home Assistant questions.  An excellent place to research and ask general help questions is at https://community.home-assistant.io/  - I'll try to help anyway though.  😄 

I wonder if this is a cache issue for you again.  If you scroll to the top of the [link I sent you before](https://github.com/PRProd/HA-Firemote/wiki/Force-a-Refresh) you can find some deeper instructions on clearing your browser cache.  When I just tried to make a helper, the dialog box disappears like you described, but I do see the new helper in the list.  I don't suppose your new helper is showing up, and you just need to scroll down to see it??

That's about all I can guess.  Sorry I can't be more helpful...

-Doug

**Reply by aughtago** [reply]
oops , told you im stupid tonight. ill try that thanks

**Reply by PRProd** [reply]
It's totally OK!  Hopefully you can get it all sorted out.  😄

**Reply by aughtago** [reply]
you were right, it was a cache refresh (only required refreshing my browser) then i got like 14 helpers from all the assumed failed attempts i made to create the same helper. thank you again for your help, even though it was beyond your pay grade (but not beyond your skills) . thank you

**Reply by PRProd** [reply]
Wow!  Cache was just being mean to you all night!

**Reply by aughtago** [reply]
ya, have learned my lesson. i am now trying to add shelly devices which I have found tend to be very fickle. i refreshed my browser routinely to see if the device appears.
